packages/lesswrong/server/callbacks/userCallbacks.ts

```diff
@@ -195,3 +198,65 @@ getCollectionHooks("Users").editSync.add(async function usersEditCheckEmail (mod
   }
   return modifier;
 });
+
+getCollectionHooks("Users").editAsync.add(async function subscribeToForumDigest (newUser: DbUser, oldUser: DbUser) {
+  if (
+    isAnyTest ||
+    forumTypeSetting.get() !== 'EAForum' ||
+    newUser.subscribedToDigest === oldUser.subscribedToDigest
+  ) {
+    return;
+  }
+
+  const mailchimpAPIKey = mailchimpAPIKeySetting.get();
+  const mailchimpForumDigestListId = mailchimpForumDigestListIdSetting.get();
+  const { lat: latitude, lng: longitude } = userGetLocation(newUser);
+  const status = newUser.subscribedToDigest ? 'subscribed' : 'unsubscribed';   
+  await fetch(`https://us8.api.mailchimp.com/3.0/lists/${mailchimpForumDigestListId}/members`, {
```

Comment:
> Related to the above, currently editAsync hooks are awaited. I claim we might want to not await this fetch (void fetch(...)) to avoid holding up the show.

Thanks, fixed in https://github.com/centre-for-effective-altruism/EAForum/pull/176/commits/25f2a3f4c9d046a385e73d784e69230d6f54e8d4